### PR TITLE
update k8s docs

### PR DESCRIPTION
**Pull request by corneil**
References: #5406

### CODE REVIEW

**Review by cppwfs — APPROVED** [review]
LGTM

File: spring-cloud-dataflow-docs/src/main/asciidoc/configuration-kubernetes.adoc [hunk 1]
```diff
@@ -839,6 +863,15 @@ deployer.<application>.kubernetes.cron.ttlSecondsAfterFinished=86400
 ----
 ====
 
+The following example shows how you can individually configure application jobs:
+
+====
+[source,options=nowrap]
+----
+deployer.<application>.kubernetes.ttlSecondsAfterFinished=86400
```
Comment (onobc) [inline spring-cloud-dataflow-docs/src/main/asciidoc/configuration-kubernetes.adoc:871]: What is this related to?

Comment (corneil) [reply spring-cloud-dataflow-docs/src/main/asciidoc/configuration-kubernetes.adoc:871]: There was an addition done a while ago for cron.ttlSecondsAfterFinished and this was added but not documented. #5406

Comment (onobc) [reply spring-cloud-dataflow-docs/src/main/asciidoc/configuration-kubernetes.adoc:871]: Got it. Even though it is small I would probably put this in its own commit in this PR for later find/perusal. I think it is not significant enough to change now though.
